src/main/java/io/fabric8/jenkins/openshiftsync/ConfigMapWatcher.java

```diff
@@ -190,6 +196,46 @@ private boolean containsSlave(ConfigMap configMap) {
                 podTemplate = xStream2.fromXML(entry.getValue());
 
                 if( podTemplate instanceof PodTemplate ) {
+                    PodTemplate pt = (PodTemplate)podTemplate;
+                    
+                    // if the image field is and imagestream related ref, without the internal docker registry, prepend it
+                    String image = pt.getImage();
+                    try {
+                        if (!OpenShiftUtils.hasRegistry(image)) {
```

Comment:
not a fan of tweaking the setting and more parsing of the string and adding an xml element wouldn't work / be hard to do

I like the annotation on the config map, where it can be a whitespace separated list of config map data keys, where we do the conversion for pod template entries in the list, and leave the ones out of the list alone